package bankSystem;

public class TestError {
	public static void main(String[] args) {
		double number1 = 0;
		int number2 = 0;
		try {
			number2 = number1;
		} catch (RuntimeException exception) {
			System.out.println("Erro: " + exception);
		}
		System.out.println(result);
	}
}